libraries/math/src/hierarchical-clustering/wasm/clustering-worker-creator.ts

```diff
@@ -0,0 +1,20 @@
+import {ClusterMatrix} from '../types';
+
+/** Calculates Cluster matrix based on distance matrix reduces array. i.e. the top right triangle of the matrix
+ * turned into the array.
+ * @param distMatArray - distance matrix reduces array.
+ * @param n - dimension of the distance matrix, i.e. side length.
+ * @param methodCode - method code for the clustering algorithm.
```

Comment:
I think that some notes about methodCode should be added.
For example, 0 - cool method, 1 - better method, 2 - the best method, etc.